apps/pdl-forvalter/src/main/java/no/nav/pdl/forvalter/utils/TilfeldigKommuneService.java

```diff
@@ -0,0 +1,35 @@
+package no.nav.pdl.forvalter.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.security.SecureRandom;
+import java.util.Properties;
+
+@Slf4j
+@Service
+public class TilfeldigKommuneService {
+
+    private static final Properties kommuner = new Properties();
+
+    static {
+        var resource = new ClassPathResource("kommuner/kommuner.yaml");
```

Comment:
Bruke et rammeverk for å lese yaml?

EKs jackson-dataformat-yaml som er tilgjenelig fra i spring boot

implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'